github4s/jvm/src/test/scala/github4s/utils/MockGithubApiServer.scala

```diff
@@ -637,7 +673,7 @@ trait MockGithubApiServer extends MockServerService with FakeResponses with Test
       request
         .withMethod("POST")
         .withPath(s"/repos/$validRepoOwner/$validRepoName/issues")
-        .withHeader(not("Authorization")))
+        .withHeader("Authorization", tokenHeader))
```

Comment:
Yeah, same as above: the difference here are strictly correct but don't reflect how I actually just added a block of text, instead trying to find the way to represent that with the minimum of changes.